lib/msal-common/src/authority/Authority.ts

```diff
@@ -309,7 +311,7 @@ export class Authority {
 
             metadataEntity.updateEndpointMetadata(metadata, true);
             return AuthorityMetadataSource.NETWORK;
-        }    
```

Comment:
In general refactoring/cleanup is always appreciated, but I'd recommend you create a separate PR just for stylistic or refactoring changes.  It's easier to review changes when they're broken apart and the review can either focus on ensuring there are no functional changes (for cleanup/refactoring) or the specific functional changes otherwise.